lib/mongoose_push_web/endpoint.ex

```diff
@@ -0,0 +1,41 @@
+defmodule MongoosePushWeb.Endpoint do
+  use Phoenix.Endpoint, otp_app: :mongoose_push
+
+  # The session will be stored in the cookie and signed,
+  # this means its contents can be read but not tampered with.
+  # Set :encryption_salt if you would also like to encrypt it.
+  @session_options [
+    store: :cookie,
+    key: "_mongoose_push_key",
+    signing_salt: "URIBGSwL"
+  ]
```

Comment:
From what I understand, we don't need this in MongoosePush since there is no session needed.

Reply:
@rslota what do you think?

Reply:
Yeah, it seems that we can remove this along with entire `Plug.Session`.